cmd/config/docs/api-conventions/manifest-annotations.md

```diff
@@ -0,0 +1,21 @@
+# Manifest Annotations
+
+This document lists the annotations that can be declared in resource manifests.
+
+### `config.kubernetes.io/local-config`
+
+A value of `"true"` for this annotation declares that the configuration is to
+local tools rather than a remote Resource. e.g. The `Kustomization` config in a
+`kustomization.yaml` **SHOULD** contain this annotation so that tools know it is
```

Comment:
I'm not sure about this example.  I'm not sure I've ever seen a `Kustomization` with this annotation. Many don't even have proper type meta. We could (and probably should if this is our stance) make `kustomize fix` add it, but I feel like that would be low-value noise for most of our users. Maybe an example focussed on KRM-style transformer configuration would work? Especially in an example like [this one](https://github.com/kubernetes/enhancements/tree/master/keps/sig-cli/2299-kustomize-plugin-composition/example/kustomization) where a transformer config starts out as a resource, strongly recommending it have the annotation makes a lot of sense.

Reply:
I agree, well-known resource kinds such as Kustomization and Kptfile should not require this explicit annotation. I removed the example.